src/main/java/MultiBracketValidation.java

```diff
@@ -0,0 +1,52 @@
+public class MultiBracketValidation {
```

Comment:
The ideal solution for this problem is to push all the brackets into a stack and as you come across a closing bracket, you should peek to see if the top is a matching closing bracket.